### ISSUE TITLE
Not able to release parking brake

### ISSUE THREAD

**Issue by ChrizzyO** [flybywiresim/a32nx#5305]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-07-12T16:47:00+00:00",
  "ref": "master",
  "sha": "27c0c2a532eeae67681ddce877ec042dce03cdbe",
  "actor": "RichardPilbery",
  "event_name": "manual"
}
```

### Describe the bug

Not able to release the parking brake - neither with mouse nor button on honeycomb yoke.

### Expected behavior

Should be released when releasing with mouse or button

### Steps to reproduce

Install latest dev version

### References (optional)

Went thruw several steps trying to resolve this with the help of your dev people on your discord. 
Set Win10 region to beta
Unplugging all devices
Emptied community folder
Unassigned p-brake button

### Discord Username (optional)

ChrisO#0319

**Comment by ImenesFBW** [comment]
Can confirm the support team gave it their best shot, and we were not able to solve the problem.

**Comment by ChrizzyO** [comment]
It´s finally solved! Benjozork came with the solving question in my other bug report!! I did in fact have a simconnect.cfg in my Documents folder that I removed and voila - everything is working again! Not sure how that simconnect.cfg file came there though... My guess is that this solution can be helpful for others as well. Some might take a backup of this file and place in Documents folders etc. and stand clueless if the aircraft systems stops working.
